test.js

```
@@ -1 +1,8 @@
 console.log(helloworld)
+
+let x = 6
+const x = 5
```

Biome (1.9.4)

[error] Shouldn't redeclare 'x'. Consider to delete it or rename it.

'x' is defined here:


(lint/suspicious/noRedeclare)